common/db.go

```diff
@@ -111,9 +114,26 @@ var schema = `
 		sig BLOB,
 		sessionID STRING
 	);
-
 	CREATE INDEX IF NOT EXISTS idx_winningtickets_sessionid ON winningTickets(sessionID);
 
+	CREATE TABLE IF NOT EXISTS ticketQueue (
+		createdAt DATETIME DEFAULT CURRENT_TIMESTAMP,
+		sender STRING,
+		recipient STRING,
+		faceValue BLOB,
+		winProb BLOB,
+		senderNonce INTEGER,
+		recipientRand BLOB,
+		recipientRandHash STRING,
+		sig BLOB PRIMARY KEY,
+		creationRound int64,
+		creationRoundBlockHash STRING,
+		paramsExpirationBlock int64,
+		submitted BOOLEAN
```

Comment:
Could this be a timestamp instead as suggested by Josh i.e. `submittedAt`  since it not only can be used to filter out tickets that have not been submitted, but it also captures additional information about when the ticket was submitted?

Reply:
I don't really see any added value in doing so. Once a ticket has been submitted on-chain all this metadata is already available on more permanent resources including its block timestamp. I don't think there's a clear use case for having submission timestamps for tickets that errored out eventually either. 

The purpose of the db backed ticket queue is to not lose winning tickets making this a forensics tool seems out of scope for this issue given its still questionable usefulness in the future as opposed to external tools we already have available. 

I feel like not removing winning tickets is already a good compromise here without knowing the full usefulness of retaining historical data.

Reply:
> The purpose of the db backed ticket queue is to not lose winning tickets making this a forensics tool seems out of scope for this issue

Agreed that the forensics is not the top priority here.

However, in this case, using a timestamp captures strictly more information while still accomplishing the same goal as a boolean. Furthermore, checking if a timestamp is not null vs. checking if a boolean is true/false for the purposes of filtering out submitted tickets requires the same amount of code. Given that the amount of code required for using a timestamp is equivalent to amount of code required for using a boolean, why not use a timestamp that captures extra information that could be useful?